### ISSUE TITLE
Flat GP awards?

### ISSUE THREAD

**Issue by banpower1** [Mordonus/RaidOps#433]
Is there a way to make flat GP awards, so that it isn't a calculation based on item level or item power?

**Comment by Drutol** [comment]
Hello, you can just add GP manually via main window , or do you mean something different?

**Comment by Hysteriana** [comment]
maybe he means every item has the same GP value regardless of ilvl or
ipower since there are at least 2 different item levels per raid.

**Comment by Drutol** [comment]
That's what I initially thought , there's no problem adding that if that's the case ^^

**Comment by Hysteriana** [comment]
well.. then do it, just in case.. addon needs was more options anyway..  😈

**Comment by Drutol** [comment]
Throw more............ check-boxes?

**Comment by Hysteriana** [comment]
and...typos !

**Comment by banpower1** [comment]
As the current game stands, the difference in the item levels (ilvl 96 vs 100 and 116 vs 120 for Genetic Archives and Datascape respectively) is very minor, and often you won't find all items in both item level versions. On top of that the guild i'm running isn't very hardcore, so i was aiming at a simplistic system where 1 piece of loot is 2 GP, 1 runefragment 1 GP. But without the option to turn of scaling based on item level or item power i'm unable to use the EP/GP calculator in the addon as is. (I will still be using the rest of it for distributing loot and keeping track of loot and attendance so i can record it on the website we're running our forums on)

TL;DR: please make a "don't scale GP with item" option, so that Item Cost = Variable \* Slot value / Quality instead of Item Cost = Item Level \* Variable \* slot value / quality

**Comment by Drutol** [comment]
Even less problem with that ^^. Expect it being delivered today.

**Comment by Drutol** [comment]
![image](https://cloud.githubusercontent.com/assets/6102530/10942187/308216c2-830f-11e5-9b7c-2b5ea6eb497a.png)
Hmm?

I'll be pushing this in a few hours to curse , if you happen to check this and request something more/different it should make it with today's update :)

**Comment by Drutol** [comment]
Added v3.08
